forge-game/src/main/java/forge/game/ability/SpellAbilityEffect.java

```diff
@@ -86,7 +86,8 @@ public final String getStackDescriptionWithSubs(final Map<String, String> params
 
                     int idx = spellDesc.indexOf("(");
                     if (idx > 0) { //trim reminder text from StackDesc
-                        spellDesc = spellDesc.substring(0, spellDesc.indexOf("(") - 1);
+                        String rem = spellDesc.substring(idx - 1, spellDesc.indexOf(")") + 1);
```

Comment:
this smells like easy IndexOutOfBoundsException
was this properly tested?

Reply:
The other code was giving a bad output for this card so something needed to change

This fixed this card... but I think I see why you don't like it

Reply:
New version seems to work well
<img width="287" alt="image" src="https://github.com/Card-Forge/forge/assets/103371817/e6df4f73-309d-4193-b3a6-1def9c4517f3">